keycloak-service-module/src/androidTest/java/org/aerogear/mobile/keycloak_service_module/ExampleInstrumentedTest.java

```diff
@@ -0,0 +1,26 @@
+package org.aerogear.mobile.keycloak_service_module;
```

Comment:
See naming suggestions email thread. Suggestion was to drop `keycloak` and use `auth`
Module name seems redundant. WDYT `auth_service`?

Reply:
Keycloak is an implementation of an auth spec we have not defined.  Calling it auth seems a bit premature.

Reply:
@secondsun - Actually there is already service merged into the core. We will need to rebase this to get that in.